### ISSUE TITLE
Installing on Linux

### ISSUE THREAD

**Issue by anshulvj** [rookiepig/CrossScaleStereo#1]
Is there a readily available Linux installation package for CSSM (with cmake files etc), or do we need to create one on our own?

**Comment by rookiepig** [comment]
Hi, at that time, I worked on Windows platform so I chose to use Visual Studio. I think it is easy to write a CMakeLists.txt for this project, the only 3rd dependency is OpenCV. something like this (may not strictly follow the cmake grammar):

````
# add all the srouces and include dir
add_src(xxxx) 
add_include
# only one executable file
add_excutable()
# link to Opencv
find_opencv()
target_link_lib(xxxx)
````

**Comment by anshulvj** [comment]
Oh, understood. Thanks, I'll check it out.

**Comment by anshulvj** [comment]
Ok. So I tried installing on Linux but I'm giving Windows another shot. So I got everything working in VS2015, on Win 8.1, and it's compiling. But when I try to run the CSPM.exe, it gives me the "The program can't run because gflags.dll is missing".

I had to separately install gflags from [here] (https://github.com/gflags/gflags/blob/master/INSTALL.md). And I point the project properties to the respective lib and bin folders of the gflags build folder.

I don't think this is a problem with your project per se, I think it's something related to my windows environment. Did you face any similar issue w.r.t. gflags? If not, what was your build environment, so that I'll see if I can replicate the same.

Thanks in advance.
Anshul
